lib/Markup/Markup.cpp

```diff
@@ -15,6 +15,9 @@
 #include "swift/AST/Comment.h"
 #include "swift/Markup/LineList.h"
 #include "swift/Markup/Markup.h"
+#if defined(__CYGWIN__)
```

Comment:
This is not CMark source problem.

There was some problem in dynamic linking with cmark, I simply chose the static link method for a workaround. I dropped this part.

I'll solve the dynamic linking problem later.